mirascope/anthropic/types.py

```diff
@@ -90,12 +91,19 @@ class BookRecommender(AnthropicCall):
     @property
     def tools(self) -> Optional[list[AnthropicTool]]:
         """Returns the tools for the 0th choice message."""
+        if not self.tool_types:
+            return None
+
         if (
-            not self.tool_types
-            or self.response.stop_reason != "stop_sequence"
+            self.response.stop_reason != "stop_sequence"
             or self.response.stop_sequence != "</function_calls>"
         ):
-            return None
+            raise RuntimeError(
+                "Generation stopped before `</function_calls>` stop sequence. "
+                "This is likely due to a limit on output tokens that is too low. "
+                "Note that this could also indicate no tool is beind called, so we "
```

Comment:
would be nice to have a copy and pastable command that people can use to check. something like call.response.stop_reason or something.

Reply:
I should just include it in the text. good call